### ISSUE TITLE
AttributeError raised when a DRF serializer has a ListField

### ISSUE THREAD

**Issue by jeremystretch** [PaulGilmartin/graph_wrap#14]
I've noticed an issue processing DRF serializers which have a [ListField](https://www.django-rest-framework.org/api-guide/fields/#listfield). (We use these primarily to reflect array fields on a model.) It looks like `ApiTransformer._get_nested_serializer()` treats ListFields as nested serializers, because they have a `child` attribute which specifies the child field. However, the child field of course doesn't have a `fields` attribute.

https://github.com/PaulGilmartin/graph_wrap/blob/d26f100c99d7091595b856dce78e4559abef42e5/graph_wrap/django_rest_framework/api_transformer.py#L48-L53

I was able to work around this by simply extending `_get_nested_serializer()` to ignore instances of `rest_framework.fields.ListField`. I _think_ this makes sense, but let me know. If so, I'm happy to submit a PR for the change. Thanks!

**Comment by PaulGilmartin** [comment]
Thanks for raising! I'll try to have a look at the issue and your suggested solution some time tonight if I get the chance :)

**Comment by PaulGilmartin** [comment]
@jeremystretch Thanks again for finding this. Your solution was a good one I believe. I've just gone ahead and pushed a fix myself. In addition to your suggestion, I also made the same fix for DictField and HStoreField.

In addition to fixing that bug, I also made ListValuedFieldTransformer more intelligent - in the previous state it was always resolving the schema type as `[GenericScalar]`. Now it inspects the `child` attribute of the list field to get the correct internal type (so for example if you have a ListField with a child `CharField`, the schema type will be of the form '[String]`.

Let me know if this fixes your issue!
